src/System.Collections.Immutable/tests/ImmutableArrayBuilderTest.cs

```diff
@@ -183,9 +183,9 @@ public void AddRangeDerivedBuilder()
         public void Contains()
         {
             var builder = new ImmutableArray<int>.Builder();
-            Assert.False(builder.Contains(1));
+            Assert.DoesNotContain(1, builder);
             builder.Add(1);
-            Assert.True(builder.Contains(1));
+            Assert.Contains(1, builder);
         }
```

Comment:
This is a test for the Contains method; it should continue to use Contains directly.